US-CBP/cbp-theme
# fix for IE browser issue recognition of object.assign() + jQuery vulnerability fix + minor updates

#### What's this PR do?

There are a few different updates within this:

- fix for an existing IE browser compatibility when using the object.assign() method by installing the "@babel/plugin-transform-object-assign": "7.2.0", and updating babel's configuration settings within the "rollup-config-helper.js" file by adding the new plugin option below it's preset options. 

- Updated jQuery core to 3.4.1 version to fix the vulnerability issues dealing with prototype pollution

- Fixed some of the syntax issues within the index.html file

- added css rule providing a margin-bottom to all forms, to help give space between them

#### Where should the reviewer start?

Can compare the current live kitchen sink with the updated version within the IE browser, and look at certain components that are having issues, ex: DatePicker, Select2, Selectize, etc..

#### How should this be manually tested?

using IE browser to view functionality of components with the kitchen sink and running audit to see if the jQuery vulnerability is handled with version 3.4.1.

#### Any background context you want to provide?

We wanted to make sure and provide all fixes for IE browser after updating to Babel 7, and continue to improve upon the overall theme.

#### What are the relevant tickets?

n/a

#### Screenshots (if appropriate)

n/a

#### Questions:
- Is there a blog post?
n/a

- Does the knowledge base need an update?

maybe
